app/models/work.rb

```diff
@@ -0,0 +1,37 @@
+class Work < ApplicationRecord
+  validates :category, presence: true
+  validates :title, presence: true, uniqueness: true
+
+  has_many :votes, dependent: :destroy
+
+  def self.sort_by_category(category)
+    works_sorted_by_category = []
+    
+    Work.all.each do |work|
+      if work.category == category
+       works_sorted_by_category << work
+      end
+    end
```

Comment:
Consider this refactor: using `works_sorted_by_category = Work.where(category: category)`

```ruby
  def self.sort_by_category(category)
    works_sorted_by_category = []
    works_sorted_by_category = Work.where(category: category)
    return works_sorted_by_category
  end
```

You'd have to update your test to not expect this to be an `Array` though